Documentation/ApiOverview/Backend/JavaScript/Modules/Modals.rst

```diff
@@ -27,91 +30,118 @@ The API provides only two public methods:
 #. :code:`TYPO3.Modal.dismiss()`
 
 
-Modal Settings
+Modal settings
 --------------
 
-========= =============== ============ ===========
-Name      DataType        Mandatory    Description
-========= =============== ============ ===========
-title     string          Yes          The title displayed in the modal
-content   string|jQuery   Yes          The content displayed in the modal
-severity  int                          Represents the severity of a modal. Please see :code:`TYPO3.Severity`. Default is :code:`TYPO3.Severity.info`.
-buttons   object[]                     Actions rendered into the modal footer. If empty, the footer is not rendered. See table below.
-========= =============== ============ ===========
+..  confval:: title
 
+    :Required: true
+    :type: string
 
-Button Settings
+    The title displayed in the modal
+
+..  confval:: content
+
+    :Required: true
+    :type: string|jQuery
+
+    The content displayed in the modal
+
+..  confval:: severity
+
+    :type: int
+    :Default: :code:`TYPO3.Severity.info`
+
+    Represents the severity of a modal. Please see :code:`TYPO3.Severity`.
+
+..  confval:: buttons
+
+    :type: object[]
+
+    Actions rendered into the modal footer. If empty, the footer
+    is not rendered. See table below.
```

Comment:
There is no table below. Can we link to the next section?

Reply:
done